src/universal/redux/makeReducer.js

```diff
@@ -19,7 +19,7 @@ const formPlugin = {
 };
 
 const appReducers = {
-  auth,
+  [DEFAULT_AUTH_REDUCER_NAME]: auth,
```

Comment:
We don't *need* this to be dynamic. But, we wrote AuthEngine to have a parametric reducer name. To keep things testable, I had to wire it all the way through.

If we want to remove this, I suggest we remove the parametric reducer name from AuthEngine (and the rest of the project) in a separate PR.

I'm ambivalent.